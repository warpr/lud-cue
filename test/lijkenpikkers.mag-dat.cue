REM COMMENT l√ªd cue v1
REM MUSICBRAINZ_ALBUM_ID 2fddaa2c-61eb-46d4-abf2-1ba80c59e487
REM LUD_DURATION_IN_SECONDS 2830.653
TITLE "Mag dat?"
PERFORMER "Lijkenpikkers"
FILE "disc01a.m4a" WAVE
  TRACK 01 AUDIO
    TITLE "Heb je al gehoord"
    PERFORMER "Lijkenpikkers"
    INDEX 01 0:00:00
  TRACK 02 AUDIO
    TITLE "Op je hoeden"
    PERFORMER "Lijkenpikkers"
    INDEX 01 1:59:70
  TRACK 03 AUDIO
    TITLE "Hallisunaties"
    PERFORMER "Lijkenpikkers"
    INDEX 01 7:35:29
  TRACK 04 AUDIO
    TITLE "Eng doen"
    PERFORMER "Lijkenpikkers"
    INDEX 01 12:32:18
  TRACK 05 AUDIO
    TITLE "Het gaat niet over rozen"
    PERFORMER "Lijkenpikkers"
    INDEX 01 16:55:06
  TRACK 06 AUDIO
    TITLE "De slechtste"
    PERFORMER "Lijkenpikkers"
    INDEX 01 21:19:35
FILE "disc01b.m4a" WAVE
  TRACK 07 AUDIO
    TITLE "Klappen"
    PERFORMER "Lijkenpikkers"
    INDEX 01 0:00:00
  TRACK 08 AUDIO
    TITLE "Wijven"
    PERFORMER "Lijkenpikkers"
    INDEX 01 6:07:67
  TRACK 09 AUDIO
    TITLE "Syndroom van touret"
    PERFORMER "Lijkenpikkers"
    INDEX 01 13:02:52
  TRACK 10 AUDIO
    TITLE "Laat me met rust"
    PERFORMER "Lijkenpikkers"
    INDEX 01 15:35:23
  TRACK 11 AUDIO
    TITLE "Het doek valt"
    PERFORMER "Lijkenpikkers"
    INDEX 01 20:00:04
