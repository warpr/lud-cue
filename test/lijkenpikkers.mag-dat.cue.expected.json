{
    "artist": "Lijkenpikkers",
    "comments": ["l√ªd cue v1"],
    "title": "Mag dat?",
    "duration": 2830.653,
    "barcode": null,
    "pos": 1,
    "mbid": "https://musicbrainz.org/release/2fddaa2c-61eb-46d4-abf2-1ba80c59e487#_",
    "files": [
        {
            "filename": "disc01a.m4a",
            "format": "WAVE",
            "tracks": [
                {
                    "artist": "Lijkenpikkers",
                    "title": "Heb je al gehoord",
                    "duration": 119.93333333333334,
                    "pos": 1,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "0:00:00",
                        "seconds": 0
                    }
                },
                {
                    "artist": "Lijkenpikkers",
                    "title": "Op je hoeden",
                    "duration": 335.4533333333333,
                    "pos": 2,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "1:59:70",
                        "seconds": 119.93333333333334
                    }
                },
                {
                    "artist": "Lijkenpikkers",
                    "title": "Hallisunaties",
                    "duration": 296.85333333333335,
                    "pos": 3,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "7:35:29",
                        "seconds": 455.38666666666666
                    }
                },
                {
                    "artist": "Lijkenpikkers",
                    "title": "Eng doen",
                    "duration": 262.84000000000003,
                    "pos": 4,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "12:32:18",
                        "seconds": 752.24
                    }
                },
                {
                    "artist": "Lijkenpikkers",
                    "title": "Het gaat niet over rozen",
                    "duration": 264.38666666666666,
                    "pos": 5,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "16:55:06",
                        "seconds": 1015.08
                    }
                },
                {
                    "artist": "Lijkenpikkers",
                    "title": "De slechtste",
                    "duration": null,
                    "pos": 6,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "21:19:35",
                        "seconds": 1279.4666666666667
                    }
                }
            ]
        },
        {
            "filename": "disc01b.m4a",
            "format": "WAVE",
            "tracks": [
                {
                    "artist": "Lijkenpikkers",
                    "title": "Klappen",
                    "duration": 367.8933333333333,
                    "pos": 7,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "0:00:00",
                        "seconds": 0
                    }
                },
                {
                    "artist": "Lijkenpikkers",
                    "title": "Wijven",
                    "duration": 414.80000000000007,
                    "pos": 8,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "6:07:67",
                        "seconds": 367.8933333333333
                    }
                },
                {
                    "artist": "Lijkenpikkers",
                    "title": "Syndroom van touret",
                    "duration": 152.61333333333323,
                    "pos": 9,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "13:02:52",
                        "seconds": 782.6933333333334
                    }
                },
                {
                    "artist": "Lijkenpikkers",
                    "title": "Laat me met rust",
                    "duration": 264.74666666666667,
                    "pos": 10,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "15:35:23",
                        "seconds": 935.3066666666666
                    }
                },
                {
                    "artist": "Lijkenpikkers",
                    "title": "Het doek valt",
                    "duration": null,
                    "pos": 11,
                    "pregap": null,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "20:00:04",
                        "seconds": 1200.0533333333333
                    }
                }
            ]
        }
    ]
}
